Source/Core/Core/Boot/ElfReader.h

```diff
@@ -39,7 +39,7 @@ class ElfReader
   ElfMachine GetMachine() const { return (ElfMachine)(header->e_machine); }
   u32 GetEntryPoint() const { return entryPoint; }
   u32 GetFlags() const { return (u32)(header->e_flags); }
-  bool LoadIntoMemory();
+  bool LoadIntoMemory(bool only_in_mem1 = true);
```

Comment:
I don't ever see this being called with `false`? Part of the original code won't be run `p_vaddr >= 0x1800000`.
If this is still correct, could you possibly refresh my mind on why `0x1800000` is so special in the mem1 case (and/or place a comment somewhere, to document its purpose)?